.github/workflows/ci.yaml

```
@@ -24,15 +24,36 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
+    # Generate lock file early to ensure it exists for cache key
+    - name: Generate uv.lock if missing
+      run: |
+        if [ ! -f uv.lock ]; then
+          echo "Generating uv.lock..."
+          uv pip compile pyproject.toml --all-extras
+        fi
+
+    # Debug: List files after lock generation to ensure uv.lock exists
+    - name: Debug: List files after lock generation
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 36: mapping values are not allowed in this context

(syntax-check)